src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  error: boolean;
+  onHandleError: (arg: boolean) => void;
+}
+
+export const Notification: React.FC<Props> = ({ error, onHandleError }) => {
+  const hendleError = () => {
+    onHandleError(false);
+  };
+
+  return (
+    <div
+      className={
+        classNames(
+          'notification is-danger is-light has-text-weight-normal', {
+            hidden: !error,
+          },
+        )
+      }
+    >
+      <button
+        type="button"
+        className="delete"
+        onClick={hendleError}
```

Comment:
maybe can be better rename that func like `closeNotification`, that function only close notification error